### ISSUE TITLE
Operands could not be broadcast together with shapes (512,512) (306,220)

### ISSUE THREAD

**Issue by lianqing11** [backtime92/CRAFT-Reimplementation#6]
Hi author, thanks for your open source. 

When I train the icdar15 dataset, I met this problem: operands could not be broadcast together with shapes (512,512) (306,220). Do you also met this problem? And do you know which line of code causes this warning?

**Comment by backtime92** [comment]
@lianqing11 yes, there are some box width or height smaller than 1, so in four_point_transform function the warped with size of (512 * 512) because of the cv2.warpPerspective.
And I think you can ignore it， because it did not influence a lot for results. Also, I will solve it as soon as possible. Or if you have any advice,please contact me.
